src/main/resources/fileTemplates/internal/ComposeScreen.kt.ft

```diff
@@ -13,7 +15,7 @@ fun ${NAME}Screen(
 
 @Composable
 @Preview(name = "${NAME}")
-private fun ${NAME}ScreenPreview() {
+private fun ${NAME}ScreenPreview(#if (${USE_PREVIEW_PARAMETER_PROVIDER} == "true") @PreviewParameter(${NAME}StatePreviewParameterProvider::class) ${NAME}: ${NAME}State #end) {
     ${NAME}Screen(
         state = ${NAME}State(),
         actions = ${NAME}Actions()
```

Comment:
Think this would be better to have as a separate file rather than part of the Contract, would you mind extracting this?

Reply:
https://github.com/levinzonr/jetpack-compose-ui-arch-plugin/pull/20/commits/af5afeff9c6879f9aa8fd291169999222e7d32ac

I've tried making a change to this commit!